config.js

```diff
@@ -1 +1 @@
-module.exports.githubKey = // Add your github auth token here
+module.exports.githubKey = ""
```

Comment:
Yeah for some reason the gitignore didnt ignore config.js and the token was pushed